src/librustc_codegen_ssa/callee.rs

```diff
@@ -18,6 +18,23 @@ pub fn resolve_and_get_fn<'tcx, Cx: CodegenMethods<'tcx>>(
     )
 }
 
+pub fn resolve_and_get_fn_for_ptr<'tcx,
```

Comment:
I'm not even sure why these free functions even exist anymore.

Reply:
Want me to inline this usage, inline this usage and the others, or leave as is?

Reply:
Maybe open an issue about inlining all of these functions? This module largely shouldn't exist anymore (cc @nagisa @rkruppe).

Reply:
https://github.com/rust-lang/rust/issues/65271